content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -447,3 +447,122 @@ Verbosity | Description
 * Also read [kubectl Usage Conventions](/docs/reference/kubectl/conventions/) to understand how to use kubectl in reusable scripts.
 
 * See more community [kubectl cheatsheets](https://github.com/dennyzhang/cheatsheet-kubernetes-A4).
+
+
+# Creating yaml manifests with kubectl
+
+# Creating deployment my-dep in yaml file foo2.yaml and listen on port 5710
+
+kubectl create deployment my-dep --image=busybox --port=5701 --dry-run=none -o yaml >foo2.yaml
+
+# Creating secret for docker-registry
+
+kubectl create secret docker-registry foo --dry-run="none" --docker-username=foo --docker-password=foo --docker-server=127.0.0.1 -o yaml >foo3.yaml
```

Comment:
````suggestion
## Creating yaml manifests with kubectl
```bash
# Creating deployment my-dep in yaml file foo2.yaml and listen on port 5710
kubectl create deployment my-dep --image=busybox --port=5701 --dry-run=none -o yaml >foo2.yaml

# Creating secret for docker-registry
kubectl create secret docker-registry foo --dry-run="none" --docker-username=foo --docker-password=foo --docker-server=127.0.0.1 -o yaml >foo3.yaml
```
````